portal-web/test/functional/com/liferay/portalweb/macros/headless/objects/ObjectDefinitionAPI.macro

```diff
@@ -175,6 +175,22 @@ definition {
 		return "${response}";
 	}
 
+	macro _deleteSubjectEntry {
```

Comment:
maybe here we can think of something more generic, like, creating a deletion for all Object entries. Look how it is done with ObjectEntry creation [here](https://github.com/liferay-frontend/liferay-portal/blob/7902316eb4c50a595d5a4b555a6e14d215bfa004/portal-web/test/functional/com/liferay/portalweb/macros/headless/objects/ObjectDefinitionAPI.macro#L115) Try it and if you have any doubts or questions let me know through Slack! :)

Reply:
For this i edit macro to deleteObjectEntry and rewrite detials